src/libraries/System.Globalization/tests/DateTimeFormatInfo/DateTimeFormatInfoAbbreviatedMonthGenitiveNames.cs

```diff
@@ -110,6 +110,10 @@ public void TestAbbreviatedGenitiveNamesWithAllCultures()
                     }
                 }
 
+                // TODO: Investigate why Browser's ICU fails on ja (ja-JP) locale here.
```

Comment:
please file an issue for tracking this

Reply:
ah I see there's already https://github.com/dotnet/runtime/issues/39285.

Reply:
@akoeplinger I've already fixed it in https://github.com/dotnet/icu/pull/9 🙂